### PR TITLE
issue-85 Simplified Carousel Block

### LINKED ISSUE #85: Simplify carousel block mark up
Reference page
https://main--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/zeiss-trumpf-und-fraunhofer-mit-deutschem-zukunftspreis-ausgezeichnet

Current Markup
https://github.com/hlxsites/zeiss/blob/ac0aa2f6a3f3ddf556aa7b89a8489fe0109ebf45/blocks/carousel/carousel.js#L5-L82

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #85 

Test URLs:
- Before: https://main--zeiss--hlxsites.hlx.live/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/quality-supplier-award-2019-en-only
- After: https://issue-85--zeiss--hlxsites.hlx.live/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/quality-supplier-award-2019-en-only

### CODE REVIEW

**Comment by jindaliiita** [comment]
Will add test cases

File: blocks/carousel/carousel.js [hunk 1]
```diff
@@ -1,103 +1,130 @@
...
+  paginationBullet.style.width = `${(100 / props.items.length)}%`;
+
+  slideShowBar.appendChild(paginationBullet);
+  slideShowWrapper.appendChild(slideShowBar);
+
+  carouselDiv.appendChild(slideShowWrapper);
+}
+
+function template(props) {
+  const carouselDiv = document.createElement('div');
```
Comment (sdmcraft) [inline blocks/carousel/carousel.js:54]: Not sure if we need this additional nesting. Are there additional styles needed here or can the carousel block itself become the grid container?

**Review by sdmcraft — COMMENTED** [review]
Thanks @jindaliiita . Looks good. Added a few comments.

File: blocks/carousel/carousel.js (hunk 1 above)
Comment (jindaliiita) [reply blocks/carousel/carousel.js:54]: I think yes, will check and update accordingly

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**